pyproject.toml

```diff
@@ -13,12 +13,13 @@ readme = "README.pypi.md"
 version = "2.5.19"
 
 [tool.poetry.dependencies]
+python = ">=3.8.1,<3.12"
```

Comment:
This will keep moving on some of us' editor formatters that have auto-sort dependencies enabled 😅

Reply:
lol I've been meaning to add something to the readme to install the poetry auto-sort plugin so we all have the same auto-sorter 😂